verification.py

```diff
@@ -5504,6 +5505,182 @@ def test_simulate_from_demography(self):
             pyplot.close("all")
 
 
+class SimulateAboveRoot(Test):
+
+    colors = list(mcolors.TABLEAU_COLORS.values())
+
+    def run_above_root_stats(self, **kwargs):
+        df_list = []
+
```

Comment:
Should probably take a copy of kwargs here to avoid updating the paramter.

```
kwargs = kwargs.copy()
```
is fine.